apps/web/app/components/PTBRRolePage.tsx

```diff
@@ -58,33 +58,39 @@ export const PTBRRolePage = ({ role }) => {
   }, [])
 
   const jobDetails = useMemo(() => {
-    return [
+    const details = [
       {
         icon: <Laptop className="text-indigo-600" />,
-        label: 'Tipo de Trabalho',
-        value: 'Remoto',
+        label: 'Work Type',
+        value: 'Remote',
       },
       {
         icon: <MapPin className="text-indigo-600" />,
-        label: 'Localização',
+        label: 'Location',
         value: role.country,
       },
       {
         icon: <AlarmClock className="text-indigo-600" />,
-        label: 'Tipo de Contrato',
+        label: 'Job Type',
         value: 'Full-time',
       },
-      {
-        icon: <DollarSign className="text-indigo-600" />,
-        label: 'Salário',
-        value: role.salary || '',
-      },
-      {
-        icon: <Calendar className="text-indigo-600" />,
-        label: 'Publicado em',
-        value: formatDate(role.updatedAt, 'Portuguese'),
-      },
     ]
+
+    if (role.salary) {
+      details.push({
+        icon: <DollarSign className="text-indigo-600" />,
+        label: 'Salary',
+        value: role.salary,
+      })
+    }
+
+    details.push({
+      icon: <Calendar className="text-indigo-600" />,
+      label: 'Posted On',
+      value: formatDate(role.updatedAt, 'English'),
+    })
+
+    return details
   }, [role.country, role.salary, role.updatedAt])
```

Comment:
The right way of doing this is declaring a function outside of the component, that receives a boolean salary and returns a list.

Then, inside the component you can
```
const jobDetails = useMemo(() => getDetails(salary), [salary]);
```